tests/io/test.crx.js

```diff
@@ -102,8 +104,8 @@ describe('crx.getFiles()', function() {
     };
     return myCrx.getFiles()
       .then((files) => {
-        assert.deepEqual(files, myCrx.files);
-        assert.notOk(this.fromBufferStub.called);
+        expect(files).toEqual(myCrx.files);
+        expect(this.fromBufferStub.called).toBeFalsy();
```

Comment:
Can follow up on this - I'm just mentioning it in passing, but this is the type of thing that would get better messages either using sinon.assert directly (no expect) or perhaps better still, getting some jasmine matchers working for sinon (the existing package for that is incompatible) to you can do expect syntax similar to what jest.fn has.